dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -448,9 +448,9 @@ Spdp::handle_participant_data(DCPS::MessageId id,
     // own announcement, so they don't have to wait.
     if (from != ACE_INET_Addr()) {
       if (from != disco_->spdp_rtps_relay_address()) {
-        this->tport_->write_i(guid, true, false);
+        this->tport_->write_i(guid, SpdpTransport::SEND_TO_LOCAL);
```

Comment:
We can now collapse this whole block down to `tport_->write_i(guid, from == disco_->spdp_relay_address() ? SEND_TO_RELAY : SEND_TO_LOCAL)`